### PR TITLE
Add Reaction to Post 

### PR DESCRIPTION
**Pull request by nathanelmer**
Added AddPostReaction & GetPostReactions in PostRepository.cs
Added HttpGet and Http post to PostController.cs with a route of post/PostReaction
Added GetPostReactionList & AddPostReaction in postManager.js
Added functionality in PostDetails.js to count reactions and add them

#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout ne-addToPost
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
    > Example: Add Reactions in db if you havent already(preferably use emoji type image urls). Go to post details and you should see the options for reactions underneath the current post. Should see a count of reactions in db on that post and should re-render when you add a reaction
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.

Closes #60

### CODE REVIEW

**Review by user1hunter — APPROVED** [review]
Feature works as described

**Comment by heymonicakay** [comment]
@nathanelmer can you please reference the issue related to this PR?

File: Tabloid/Repositories/PostRepository.cs [hunk 1]
```diff
@@ -131,6 +130,52 @@ OUTPUT INSERTED.ID
...
+            {
+                conn.Open();
+                using (var cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = @"INSERT INTO PostReaction (PostId, ReactionId, UserProfileId)
+                                        OUTPUT INSERTED.ID
+                                        VALUES (@postId, @reactionId, @userProfileId)";
+                    cmd.Parameters.AddWithValue("@postId", postReaction.PostId);
+                    cmd.Parameters.AddWithValue("@reactionId", postReaction.ReactionId);
+                    cmd.Parameters.AddWithValue("@userProfileId", postReaction.UserProfileId);
```
Comment (cjgochanour) [inline Tabloid/Repositories/PostRepository.cs]: switch these with DbUtils.AddParameter and you're golden

**Review by cjgochanour — CHANGES_REQUESTED** [review]
see comment

File: Tabloid/Repositories/PostRepository.cs (hunk 1 above)
Comment (nathanelmer) [reply Tabloid/Repositories/PostRepository.cs]: donezo

**Review by cjgochanour — APPROVED** [review]
Code so pretty I could marry it

**Pull request merged**